ssz_serialization/merkleization.nim

```diff
@@ -578,8 +578,8 @@ func hashTreeRootAux[T](x: T): Digest =
     hashTreeRootAux(x.bytes)
   elif T is BitList:
     const totalChunks = maxChunksCount(T, x.maxLen)
-    let contentsHash = bitListHashTreeRoot(totalChunks, BitSeq x)
-    mixInLength(contentsHash, x.len)
+    bitListHashTreeRoot(totalChunks, BitSeq x)
+      .mixInLength(x.len)
```

Comment:
Why use the dot syntax here if we avoid it elsewhere in the file?

Reply:
Will remove dots for `mixInLength` and `name` as well. The comment about the `name` is interesting, and probably applies more broadly anytime there are operations on `typedesc`.